File: Python/1036.py
import math
ipt = input().split()
a = float(ipt[0])
b = float(ipt[1])
c = float(ipt[2])
delta = b**2-4*a*c

if (round(delta,5) < 0) or (a == 0):
    print('Impossivel calcular')
else:
    deltaSqrt = math.sqrt(round(delta,5))
    print(f'R1 = {(-b + deltaSqrt)/(2*a):.5f}')
    print(f'R2 = {(-b - deltaSqrt)/(2*a):.5f}')

File: Ruby/1_Iniciante/1036.rb
ipt = gets.split()
a = ipt[0].to_f
b = ipt[1].to_f
c = ipt[2].to_f
delta = b**2-4*a*c

if delta.round(5) < 0 or a == 0
    puts "Impossivel calcular"
else
    deltaSqrt = Math.sqrt(delta.round(5))
    puts "R1 = %.5f" % ((-b + deltaSqrt)/(2*a))
    puts "R2 = %.5f" % ((-b - deltaSqrt)/(2*a))
end
